perses/dispersed/utils.py

```diff
@@ -720,12 +721,15 @@ def compute_incremental_work(self, _lambda):
         _lambda : float
             the lambda value used to update the importance sample
         """
-        old_rp = self.beta * self.context.getState(getEnergy=True).getPotentialEnergy()
+        sampler_state = SamplerState.from_context(self.context)
+        old_rp = self.thermodynamic_state.reduced_potential(sampler_state)
 
         #update thermodynamic state and context
         self.thermodynamic_state.set_alchemical_parameters(_lambda, lambda_protocol = self.lambda_protocol_class)
         self.thermodynamic_state.apply_to_context(self.context)
-        new_rp = self.beta * self.context.getState(getEnergy=True).getPotentialEnergy()
```

Comment:
why is there a difference between `self.beta * self.context.getState(getEnergy=True).getPotentialEnergy()` and `self.thermodynamic_state.reduced_potential(sampler_state)`?